astroid/brain/brain_namedtuple_enum.py

```diff
@@ -90,7 +92,12 @@ def _extract_namedtuple_arg_or_keyword(  # pylint: disable=inconsistent-return-s
     raise UseInferenceDefault()
 
 
-def infer_func_form(node, base_type, context=None, enum=False):
+def infer_func_form(
+    node: nodes.Call,
+    base_type: List[nodes.NodeNG],
```

Comment:
```suggestion
    base_type: nodes.NodeNG,
```

Reply:
Shit sorry.. Thought I double checked this 😓